README.md

```diff
@@ -20,21 +20,21 @@
 
 ## What is lakeFS
 
-lakeFS is an open source tool that transforms your object storage into a Git-like repository. It enables you to manage your data lake the way you manage your code.
+lakeFS is an open-source tool that transforms your object storage into a Git-like repository. It enables you to manage your data lake how you manage your code.
 
-With lakeFS you can build repeatable, atomic and versioned data lake operations - from complex ETL jobs to data science and analytics.
+With lakeFS you can build repeatable, atomic, and versioned data lake operations - from complex ETL jobs to data science and analytics.
 
-lakeFS supports AWS S3, Azure Blob Storage and Google Cloud Storage as its underlying storage service. It is API compatible with S3, and works seamlessly with all modern data frameworks such as Spark, Hive, AWS Athena, Presto, etc.
+lakeFS supports AWS S3, Azure Blob Storage, and Google Cloud Storage as its underlying storage service. It is API compatible with S3 and works seamlessly with all modern data frameworks such as Spark, Hive, AWS Athena, Presto, etc.
 
 
-For more information see the [official lakeFS documentation](https://docs.lakefs.io).
+For more information, see the [official lakeFS documentation](https://docs.lakefs.io).
 
 ## Capabilities
 
 **In Development**
 
-* **Experiment** - try new tools, upgrade versions, and evaluate code changes in isolation. By creating a branch of the data you get an isolated snapshot to run and compare experiments over, while others are not exposed. 
-* **Debug** - checkout specific commits in a repository’s commit history to materialize consistent, historical versions of your data. 
+* **Experiment** - try new tools, upgrade versions, and evaluate code changes in isolation. You can get an isolated snapshot to run and compare experiments by creating a new branch of the data, while others are not exposed.
+* **Debug** - check out specific commits in a repository’s commit history to materialize consistent, historical versions of your data. 
```

Comment:
```suggestion
* **Debug** - checkout specific commits in a repository’s commit history to materialize consistent, historical versions of your data. 
```
Again, while the first change is really great, the second changes semantics.  "Check out specific commits" reads as "have a look at specific commits", but here we really do mean "checkout" as a verb used by programmers: get a version out of a version control system.

Reply:
TBH that wasn't clear for me as a developer.